templates/settings.py.erb

```diff
@@ -9,7 +9,15 @@ DATABASES = {
         'PASSWORD': '<%= scope['pulpcore::postgresql_db_password'] %>',
         'HOST': '<%= scope['pulpcore::postgresql_db_host'] %>',
         'PORT': '<%= scope['pulpcore::postgresql_db_port'] %>',
-    }
+<% if scope['pulpcore::postgresql_db_ssl'] && !scope['pulpcore::postgresql_db_manage'] -%>
+	'OPTIONS': {
+            'sslmode': '<%= scope['pulpcore::postgresql_db_ssl_require'] ? 'require' : 'disable' %>',
+            'sslcert': '<%= scope['pulpcore::postgresql_db_ssl_cert'] %>',
+            'sslkey': '<%= scope['pulpcore::postgresql_db_ssl_key'] %>',
+            'sslrootcert': '<%= scope['pulpcore::postgresql_db_ssl_root_ca'] %>',
+        },
```

Comment:
Nitpick, pull this back one set of tabs to have it align when its actually rendered

Reply:
Err sorry, I think actually push it forward one set the closer I look.

Reply:
I think it's correct as is (except for the fact that the spaces in front of `OPTIONS` keep converting to a tab somehow, which I have corrected again). To be sure, I ran a puppet apply with the db unmanaged and ssl enabled on a test system and found the correct level of indentation in `/etc/pulp/settings.py`